admin_guide/upgrading_citus.rst

```diff
@@ -188,12 +207,12 @@ On the Coordinator Node
 
     SELECT setval('pg_catalog.pg_dist_shardid_seq', (SELECT MAX(shardid)+1 AS max_shard_id FROM pg_dist_shard), false);
 
+    SELECT setval('pg_catalog.pg_dist_placement_placementid_seq', (SELECT MAX(placementid)+1 AS max_placement_id FROM pg_dist_placement), false);
+
     SELECT setval('pg_catalog.pg_dist_groupid_seq', (SELECT MAX(groupid)+1 AS max_group_id FROM pg_dist_node), false);
 
     SELECT setval('pg_catalog.pg_dist_node_nodeid_seq', (SELECT MAX(nodeid)+1 AS max_node_id FROM pg_dist_node), false);
 
-    SELECT setval('pg_catalog.pg_dist_placement_placementid_seq', (SELECT MAX(placementid)+1 AS max_placement_id FROM pg_dist_placement), false);
-
     SELECT setval('pg_catalog.pg_dist_colocationid_seq', (SELECT MAX(colocationid)+1 AS max_colocation_id FROM pg_dist_colocation), false);
 
 10. Register triggers.
```

Comment:
I was a bit skeptical of steps 10 and 11 for upgrade on the workers. @onderkalaci Any issues if we register truncate triggers and set the dependency on citus extension on the workers for MX? Dependency seems fine, but do we do something special for table truncate?

Reply:
Oh, sorry that I missed this comment.

Yes, we need to create the `TRUNCATE` triggers on the workers as well for MX (because we support `TRUNCATE` from the worker nodes as well).

So, it makes sense that this step is under `On Every Node (Coordinator and workers)`.

>  but do we do something special for table truncate?

I don't see anything special for truncate other than creating the triggers.